betterrolls-swade2/scripts/settings_config.js

```diff
@@ -64,7 +64,7 @@ export class SettingsConfig extends FormApplication {
     s.id = s.key;
     s.name = game.i18n.localize(s.name);
     s.hint = game.i18n.localize(s.hint);
-    s.value = s.value !== undefined ? s.value : 
+    s.value = s.value ?? 
         (typeof s.default === 'string' && s.default.startsWith('BRSW.') ? 
```

Comment:
Sorry, I guess I wasn't clear enough in the other comment. None of this loc code is necessary. You can just use the string and if it's a string ID then localization will work.